src/udata.rs

```diff
@@ -393,7 +393,7 @@ pub mod shinigami_udata {
     use bitcoin::Txid;
     use rustreexo::accumulator::node_hash::AccumulatorHash;
     use serde::Serialize;
-    use starknet_crypto::poseidon_hash_many;
+    use starknet_crypto::{poseidon_hash_many, poseidon_hash};
```

Comment:
Just a small thing: I'm not using braced syntax for multiple imports, just one import per line.
Could you change this to:

```diff
- use starknet_crypto::{poseidon_hash_many, poseidon_hash};
+ use starknet_crypto::poseidon_hash;
+ use starknet_crypto:: poseidon_hash_many;
```

(a `cargo +nightly fmt --all` would do the trick too)

Reply:
Done!